system/classes/KO7/REST/Format/HTML.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Class for formatting REST-Response bodies as HTML
+ *
+ * @package        KO7\REST
+ *
+ * @copyright  (c) since 2016 Koseven Team
+ * @license        https://koseven.ga/LICENSE
+ */
+class KO7_REST_Format_HTML extends REST_Format {
+
+    /**
+     * Format function
+     *
+     * @throws REST_Exception
+     *
+     * @return string
+     */
+    public function format() : string
```

Comment:
~~~
$path = array_filter([
$this->_request->directory(), 
$this->_request->controller(), 
$this->_request->action(),
]);
$path = strtolower(implode(DIRECTORY_SEPARATOR, $path));
if (! KO7::find_file('views', $path)) {
   $path = dirname($path);
}
return View:factory($path,  $this->_body)->render();

Reply:
Nice way, i changed it a bit to fit into the correct order